docs/external-database.md

```diff
@@ -0,0 +1,144 @@
+# Using an external database
+
+You can use any external Postgres database instead of a local database deployment.
+Tekton Results gets its database configuration either from a Kubernetes Secret
+named `tekton-results-postgres` or from a ConfigMap with the same name.
+
+Tekton Results expects following database configurations to be provided.
+
+| Name | Secret | ConfigMap | Comments |
+| --- | --- | --- | --- |
+| Database User | `POSTGRES_USER` | `DB_USER` | default: postgres |
+| Database Password | `POSTGRES_PASSWORD` | `DB_PASSWORD` | |
```

Comment:
We should not recommend users provide sensitive database credentials in ConfigMaps.